src/strategies/masterchef-pool-balance/index.ts

```diff
@@ -146,6 +148,13 @@ export async function strategy(
   snapshot
 ) {
   const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, test, { blockTag });
+  multi.call('index', options.indexAddress, 'index');
+
+  const result = await multi.execute();
+  const index = parseFloat(formatUnits(result.index, options.indexDecimals));
+  console.log('index', index);
```

Comment:
Hi @brightiron What is the use of this change? can you remove it?

Reply:
sorry about this. I inadvertently pushed a couple files from local testing. This is reverted.